CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.5.1
+
+Changed:
+
+- `RequireFile.exists` now takes the more generic `FileSystemEntity`
```

Comment:
I consider this an 'addition' as you've added functionality that wasn't previously there. 

I think the version should be 1.6.0